### DISCUSSION TITLE
QBlocker viruses check

### DISCUSSION THREAD

**Discussion by qt-kaneko** [Homebrew/discussions#3603] [Casks]
Hi. Anyone checked [QBlocker](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/qblocker.rb) for viruses? Because, as I see [here](https://github.com/steve228uk/QBlocker/issues/34) and [here](https://github.com/steve228uk/QBlocker/issues/36), it tries to read cookies and contacts 😨. Or is it rebuilt from sources?

**Comment by p-linnane** [comment]
[Homebrew is not a first line of defense against malware.](https://docs.brew.sh/Acceptable-Casks#apps-that-bundle-malware) If enough evidence is provided of it truly being malicious, the cask will be removed.

Additionally, this application has not been updated in over 6 years and has less than 100 downloads in the last year. Odds are good it will be removed from the main cask repo anyway.
